src/utils/helpers.utils.js

```diff
@@ -103,13 +103,29 @@ export const checkDuplicateUser = async (email, username) => {
   return existingUser !== null;
 };
 
+/**
+ *
+ * @param {object} res response object
+ * @param {number} statusCode
+ * @param {string} message
```

Comment:
Good work here for adding the doc blocks

Reply:
Thank you